commercetools-models/src/main/java/io/sphere/sdk/orders/messages/DeliveryAddressSetMessage.java

```diff
@@ -0,0 +1,40 @@
+package io.sphere.sdk.orders.messages;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.sphere.sdk.messages.GenericMessageImpl;
+import io.sphere.sdk.messages.MessageDerivateHint;
+import io.sphere.sdk.models.Address;
+import io.sphere.sdk.orders.Order;
+
+import java.time.ZonedDateTime;
+
+/**
+ * his message is the result of the {@link io.sphere.sdk.orders.commands.updateactions.SetDeliveryAddress} update action.
```

Comment:
A `T` is missing here , copy and paste error from the doc 😉

Reply:
good catch 👍 
When you said T i started looking for generics 😅